main.html

```diff
@@ -24,24 +25,47 @@
                 <div class="header__burger">
                     <img src="burger.png" alt=""/>
                 </div>
+                <nav class="header-hidden__nav">
```

Comment:
is it always hidden? if it's not why should it have hidden word in class name?

Reply:
Got you, will figure this out. No, it's hidden only on mobile